libnetdata/dyn_conf/README.md

```diff
@@ -1,51 +1,74 @@
 # Netdata Dynamic Configuration
 
-Purpose of Netdata Dynamic Configuration is to allow configuration of select Netdata plugins and options trough the Netdata API and by extension by UI.
+Purpose of Netdata Dynamic Configuration is to allow configuration of select Netdata plugins and options through the
+Netdata API and by extension by UI.
 
 ## HTTP API documentation
 
 ### Summary API
+
 For summary of all jobs and their statuses (for all children that stream to parent) use the following URL:
-- `GET api/v2/job_statuses` - to get flat list of all the jobs this agent knows about
-- `GET api/v2/job_statuses?grouped` - to get hierarchical list (grouped by host/plugin/module) of all the jobs this agent knows about
+
+| Method  | Endpoint                      | Description                                                |
+|:-------:|-------------------------------|------------------------------------------------------------|
+| **GET** | `api/v2/job_statuses`         | list of Jobs                                               |
+| **GET** | `api/v2/job_statuses?grouped` | list of Jobs (hierarchical, grouped by host/plugin/module) |
 
 ### Dyncfg API
-All URLs bellow if not absolute are listed as relative to (prefix `.../`) the following URL:
-`http[s]://[your_agent_ip]/api/v2/config`
 
 ### Top level
-- `GET http[s]://[your_agent_ip]/api/v2/config` - gets the list of all dyncfg registered plugins (all plugins which called DYNCFG_ENABLE)
+
+| Method  | Endpoint         | Description                             |
+|:-------:|------------------|-----------------------------------------|
+| **GET** | `/api/v2/config` | registered Plugins (sent DYNCFG_ENABLE) |
+
 ### Plugin level
-- `GET .../[plugin_name]` - gets the plugins current configuration from the plugin using `get_plugin_config` command
-- `PUT .../[plugin_name]` - updates the plugin configuration
-- `GET .../[plugin_name]/modules` - get list of modules registered by plugin and their types
-- `GET .../[plugin_name]/schema` - gets configuration schema of the plugin
+
+| Method  | Endpoint                          | Description                  |
+|:-------:|-----------------------------------|------------------------------|
+| **GET** | `/api/v2/config/[plugin]`         | Plugin config                |
```

Comment:
@underhood I find it much easier to follow without always having in mind that "this is not a full endpoint but relative to smth", so decided to add `/api/v2/config` to all endpoints.